app/src/main/res/values/strings.xml

```diff
@@ -112,6 +114,7 @@
 	<string name="error_externalstorage_not_writable_hint">Please check if ext. storage is correctly inserted and mounted.</string>
 	<string name="error_voicerec_failed">Voice recording has failed</string>
 	<string name="error_userlayout_parsing">Error while parsing XML layout file. Please revert to default layout.</string>
+	<string name="error_no_appropriate_activity">No application able to handle gpx file was found.</string>
```

Comment:
Perhaps "...to handle GPX files..." (capital GPX, and plural files)?

Also as said elsewhere my understanding is that also can be ZIP files, so that should probably be reflected here?